src/components/progress-tracker/progress-tracker.component.js

```diff
@@ -34,6 +37,13 @@ const ProgressTracker = ({
   labelsPosition,
   ...rest
 }) => {
+  if (!deprecatedForwardRefWarnTriggered) {
```

Comment:
this will log the deprecation warning whenever the component is used. Surely we should check for whether the prop(s) are actually used and only show the warning then?

(My preference would be to have 2 separate variables and 2 separate deprecation messages, one for each prop. But I suppose it's OK if the message is logged when either of them are used, that should be understandable enough.)

Reply:
The props have default values so will always display etc, I explained why I've chosen not to add tests or refactor further in the context section above

Reply:
Yeah I still disagree, I don't think we should tell consumers "you shouldn't use this prop" when they haven't actually used it. That refactor will be needed later anyway when we come to remove the prop so why not do it now.

By all means seek a second opinion if you feel strongly enough about it.